app/handlers/ErrorHandler.scala

```diff
@@ -17,20 +17,25 @@
 package handlers
 
 import javax.inject.{ Inject, Singleton }
-import play.api.i18n.{ I18nSupport, MessagesApi }
+import play.api.i18n.{ I18nSupport, Messages, MessagesApi }
 import play.api.mvc.Request
 import play.twirl.api.Html
 import uk.gov.hmrc.play.bootstrap.frontend.http.FrontendErrorHandler
 import views.html.ErrorTemplate
+import views.html.ErrorInternalView
 
 @Singleton
 class ErrorHandler @Inject() (
   val messagesApi: MessagesApi,
-  view: ErrorTemplate
+  view: ErrorTemplate,
+  errorInternal: ErrorInternalView
 ) extends FrontendErrorHandler with I18nSupport {
 
   override def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit
     rh: Request[_]
   ): Html =
     view(pageTitle, heading, message)
+
+  override def internalServerErrorTemplate(implicit request: Request[_]): Html =
+    errorInternal()
```

Comment:
Can you remind me when is this error triggerred?

Reply:
This is server error 500, so to test it i just stopped the backend server